src/tests/fixtures/users.fixture.js

```diff
@@ -127,6 +152,7 @@ export const createUsers = async () => {
   await Users.create(createUser);
   await Users.create({ ...loggedInUser, token: loggedInToken });
   await Users.create({ ...userWithNoTrip, token: userWithNoTripToken });
+  await Users.create({ ...superAdminTestData, token: superAdminToken });
 };
 export const cleanDb = async () => {
```

Comment:
Hi @higustave-ops good work so far, but I wanted to know what did you create cleanDb to delete table while we have that function in createUsers functions?

Reply:
@adesege and @erickyvand 

* For the signup test need a fresh database table so the `cleanDb` function on `line 157` will do the job to clean it for signup test. 

* For `createUsers` function, every test may create its users. Thus a need to delete other users and create its own. That's the reason why we have both functions